cluster/examples/kubernetes/ceph/rook-operator-config.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-operator-config
+  namespace: rook-ceph
+data:
+  CSI_PLUGIN_NODE_AFFINITY: "cluster.ocs.openshift.io/openshift-storage"
```

Comment:
This is very OCS-specific. For upstream let's stick with a generic label such as `role=storage-node`